test/core/end2end/fuzzers/client_fuzzer.cc

```diff
@@ -42,7 +41,6 @@ static void dont_log(gpr_log_func_args* args) {}
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
   grpc_test_only_set_slice_hash_seed(0);
   if (squelch) gpr_set_log_function(dont_log);
-  grpc_core::testing::LeakDetector leak_detector(leak_check);
```

Comment:
Please make sure that we run the fuzzers under asan, or else we will be losing the ability to detect leaks.

Same for all of the fuzzers.

Reply:
Sounds good.

Reply:
I verified that all fuzzer tests are running under grpc_bazel_rbe_asan so it seems okay.